src/wrappers/EEPROM_TC.cpp

```diff
@@ -49,11 +59,20 @@ int32_t EEPROM_TC::eepromReadInt(uint16_t address) {
   return value;
 }
 
+/**
+ * @brief writes the value to the address and triggers an alert in DataLogger
+ *
+ * @param address
+ * @param value
+ */
 void EEPROM_TC::eepromWriteInt(uint16_t address, int32_t value) {
   assert(address >= 0);
```

Comment:
Here the assertion is not removed.

Reply:
Not for an unsigned number! I'll change it to `!= 0`.